etcd/etcd-live-http-to-https-migration.md

```diff
@@ -142,24 +132,41 @@ etcdctl member update 50da8780fd6c8919 https://172.16.0.103:2380
 etcdctl member update 81901418ed658b78 https://172.16.0.101:2380
 ```
 
-You have to run generated commands **except last one** on one of the member nodes and check etcd cluster health:
+On any one etcd cluster member, run each of the printed commands -- *except the last one*. We will change the peer URL for the last etcd cluster member only after completing any proxy configuration below.
```

Comment:
Why do we need to wait on the etcd proxy?  *Do* we need to?  We should explain the reason(s) better, or if (as I suspect), we don't actually need to delay, then just plow ahead.